ansible/roles/ensure_crc/defaults/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+# defaults file for ensure_crc
+pull_secret_path: "~/.cache/ard/pull-secret"
+crc_url: "https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/crc/latest/crc-linux-amd64.tar.xz"
+install_crc: True
+crc_cpus: 6
+crc_ram: 16384
```

Comment:
nit: these look different to RAM hardcoded for molecule vagrant driver

Reply:
yes that is intentional.

i would like to reduce these more eventually but I'm ustin a 20GB VM and then creating a 16GB crc instance inside that to leave 4GB for the host for thing like compiling the operator or whatever other things you need to do in the VM.

similarly the molecule VM will have 8 CPU so I'm only allcoating 6 to the CRC instance by default to reserve some for the host.